app/models/user.rb

```diff
@@ -16,9 +16,14 @@ def self.from_omniauth(auth)
     user.last_name  = auth["info"]["name"].split(" ").last
     user.email      = auth["info"]["email"]
     user.password   ||= rand(1000000...10000000).to_s
-    user.role       = "user"
+    user.role       = "registered"
     user.save
     user
   end
+
+  def storage_used
+    repos.count / 0.2
```

Comment:
@DavidKnott I will confirm when linking up to the frontend, but I think the output of this method must be `String` and have `%`